Cargo.lock

```diff
@@ -4409,11 +4381,7 @@ dependencies = [
 name = "solana-banking-bench"
 version = "1.10.18"
 dependencies = [
-<<<<<<< HEAD
- "clap 3.1.12",
-=======
- "clap 2.33.3",
->>>>>>> parent of 69c98236f6 (rbpf-cli: add coverage flag (#24704))
+ "clap 3.1.6",
```

Comment:
Keep the v3 upgrade from a later PR but reduce v3 back to pre-reverted-commit patch version